### PR TITLE
Pop up window

### PR DESCRIPTION
**Pull request by fatmahussein**
- used objects to store and access card data
- used Javascript events
- created pop up window modal
- I am working solo on this project as advised by **@marco**  student success team, please see attached.
[[Microverse]+Re_+I+am+really+sick.+Please+see+video+below..pdf](https://github.com/fatmahussein/My-portfolio/files/11007373/Microverse.%2BRe_%2BI%2Bam%2Breally%2Bsick.%2BPlease%2Bsee%2Bvideo%2Bbelow.pdf)

### CODE REVIEW

**Review by AlphaNtihinduka — COMMENTED** [review]
Hi @fatmahussein ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request
usb-invalid


You have submitted a project which is a pair programming milestone. Kindly reach out to your partner(s) to make a contribution in this PR. If for any reason that is officially communicated, kindly attach the proof of communication in the PR description.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.js [hunk 1]
```diff
@@ -12,37 +12,459 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
+  }
+  cardBody.appendChild(proj);
+
+  projectsContainer.appendChild(card);
 }
+window.addEventListener('resize', () => this.location.reload());
+
+// modal
+const see = document.querySelectorAll('.see-project');
+see.forEach((btn) => (btn.addEventListener('click', () => {
```
Comment (rustam-tolipov) [inline index.js]: **I can see you put great effort into this project.** 
_However, we have one thing left to do!_

- [ ] As the requirement your popup window info should relate to the project that we are opening and the data should be gotten from the array you created. We can see in your popup window only one data. Kindly manipulate the popup window data based on the array and should match the project that you opened. Thank you!

File: styles.css [hunk 2]
```diff
@@ -1755,6 +1847,204 @@
...
+  }
+
+  #live2 {
+    margin-top: 3%;
+    font-size: 14px;
+    color: #396df2;
+    padding: 2px 5px;
+  }
+
+  .buttons {
```
Comment (rustam-tolipov) [inline styles.css:2067]: - [ ] Based on your styling. It's pretty good. But one suggestion. Please add some space at the bottom of your description. Kindly see the picture:

Screenshot 2023-03-18 at 08 42 33

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @fatmahussein,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great readme_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 3]
```diff
@@ -12,37 +12,459 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
-const email = document.querySelector('#email');
-const errorMessage = document.querySelector('.error-message');
-form.addEventListener('submit', (event) => {
-  if (email.value !== email.value.toLowerCase()) {
-    errorMessage.style.visibility = 'visible';
-    errorMessage.style.position = 'inherit';
-    event.preventDefault();
+// Store project information in array
+const body = document.querySelector('body');
+const projects = [
```
Comment (Mihndim2020) [inline index.js:16]: - [ ]  As requested by the previous code reviewer, "`As the requirement your popup window info should relate to the project that we are opening and the data should be gotten from the array you created. We can see in your popup window only one data. Kindly manipulate the popup window data based on the array and should match the project that you opened. Thank you!`", please make sure each popup is populated with a value(object) from the project array, all the popups should not be having the same information. Each popup should be populated with a value from the projects array. Please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @fatmahussein, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 # Status: Requires Changes ♻️
_Check the comments under the review._

Please do consider, as requested by the previews reviewer:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**